docs/resources/kibana_action_connector.md

```diff
@@ -0,0 +1,58 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "elasticstack_kibana_action_connector Resource - terraform-provider-elasticstack"
+subcategory: ""
+description: |-
+  Creates a Kibana action connector. See https://www.elastic.co/guide/en/kibana/8.7/action-types.html
+---
+
+# elasticstack_kibana_action_connector (Resource)
+
+Creates a Kibana action connector. See https://www.elastic.co/guide/en/kibana/8.7/action-types.html
+
+## Example Usage
+
+```terraform
+provider "elasticstack" {
```

Comment:
@dimuon can you explain this resource and connector? I'm not sure what it does and why in the context of Kibana actions / connectors.

Reply:
@Kushmaro , the idea is to add support for alert connectors (API defines them as [action connectors](https://www.elastic.co/guide/en/kibana/master/actions-and-connectors-api.html)), so users can perform CRUD operations on [connectors](https://www.elastic.co/guide/en/kibana/master/action-types.html) using the stack provider. Can you please suggest proper name for the resource?

Reply:
my bad for terribly bad wording 😅, I understand this is for kibana alert connectors 
I meant, the actual resource itself, like, why is the config in jason code, as opposed to parameters?
what is "connector_type_id = ".index"" what why is it used? (seems like an internal thing) etc

Reply:
The implementation here should abstract some of the API and make it easy for a user to configure a connector just like they do in kibana UI, for example.

Reply:
AH OK, now read the API docs as well.

hmm ok, I think the ".index" really threw me off here. I'm also wondering if we shouldn't append "." by ourselves and make this just a normal value. The Terraform provider is a client like the UI. It shouldn't expose nasty internals, like us naming our things with "." in the API